novelwriter/gui/doceditor.py

```diff
@@ -1017,6 +1019,27 @@ def resizeEvent(self, event: QResizeEvent) -> None:
         super().resizeEvent(event)
         return
 
+    def inputMethodEvent(self, event: QInputMethodEvent) -> None:
+        """Handle text being input from CJK input methods."""
+        super().inputMethodEvent(event)
+        if event.commitString():
+            self.ensureCursorVisible()
+            if self._completer.isVisible():
+                rect = self.cursorRect()
+                pos = self.mapToGlobal(rect.bottomLeft())
+                self._completer.move(pos)
```

Comment:
Can you double check what the completer menu does here? As far as I can test it moves to a position on top of the text being edited, which is not intended.

Reply:
I've changed this to correspond to how the completer menu moves with regular use. Please verify.